<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorplatform</class>
 <widget class="QWidget" name="doctorplatform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listDoctors">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>104</width>
     <height>85</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMessegeInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
